Source/Scene/DracoLoader.js

```diff
@@ -177,6 +177,12 @@ define([
             return when.resolve();
         }
 
+        if (FeatureDetection.isInternetExplorer()) {
+            return when.reject({
+                message : 'Draco decoding is not supported in legacy environments.'
```

Comment:
To be a little more obvious, maybe just say "Draco decoding is not currently support in Internet Explorer"